samples/subsys/display/lvgl/prj.conf

```diff
@@ -1,4 +1,4 @@
-CONFIG_HEAP_MEM_POOL_SIZE=16384
+CONFIG_LVGL_MEM_POOL_NUMBER_BLOCKS=8
```

Comment:
good example of my previous comment where the Kconfigs switch preambles

Reply:
No, it's a different thing. The deleted line defines the global kernel heap size. The new line defines the number of heap blocks for the private LVGL heap. This is done because the whole kernel heap allocator for lvgl is now gone. The default size of a block is 2048 which results in the same pool size.